chipy_org/apps/job_board/models.py

```diff
@@ -10,11 +10,7 @@
 MAX_LENGTH = 100
 NUM_DAYS_T0_EXPIRE = 60
 
-STATUS_CHOICES = [
-    ("SU", "Submitted"),
-    ("AP", "Approved"),
-    ("RE", "Rejected"),
-]
+STATUS_CHOICES = [("SU", "Submitted"), ("AP", "Approved"), ("RE", "Rejected"), ("NA", "Filled")]
```

Comment:
NIT: this isn't formatted correctly (the project uses the black formatting). Regardless the pre-commit hook will catch it.